package.json

```diff
@@ -104,6 +104,8 @@
     "flow-bin": "^0.85.0",
     "har-validator": "^5.1.3",
     "husky": "^1.1.3",
+    "less": "^2.3.1",
```

Comment:
If anyone’s wondering why we’re not using `v3` and up, it’s because of something similar like [this](https://github.com/ant-design/ant-design/issues/7850).